docs/billings-mt.md

```diff
@@ -0,0 +1,6 @@
+# Billings
+
+- The best thing about Billings is the view from the top of the Rimrocks, a sandstone bluff that runs along the north side of the Yellowstone valley.
```

Comment:
54 F currently; for more info please see http://www.weather.com/weather/tenday/l/Billings+MT+59101:4:US